t/script/release/skip_broken_files.t

```diff
@@ -0,0 +1,42 @@
+use strict;
+use warnings;
+
+use Archive::Tar;
+use HTTP::Request;
+use LWP::UserAgent;
+use MetaCPAN::Script::Release;
+use Test::More;
+
+use Test::RequiresInternet ( 'metacpan.org' => 443 );
+
+my $ua = LWP::UserAgent->new(
+    parse_head => 0,
+    env_proxy  => 1,
+    agent      => 'metacpan',
+    timeout    => 30,
+);
+
+my $URL
+    = 'https://cpan.metacpan.org/authors/id/S/SH/SHULL/karma-0.7.0.tar.gz';
+my $tarball = 'tarball.tar.gz';
+
+my $req = HTTP::Request->new( GET => $URL );
+$req->header( Accept => '*/*' );
+my $res = $ua->request( $req, $tarball );
```

Comment:
You can use a temporary directory for the download if this is going to end up in your checkout folder.  Look at https://metacpan.org/pod/File::Temp for that.